Makefile

```diff
@@ -98,16 +104,19 @@ endif
 
 DOCKER_FILE := $(shell mktemp)
 
+
 .PHONY: all clean test
 
 all:
 	@echo Build docker image with $(IMAGE_TAG) tag
-	@cp dockerfile.template $(DOCKER_FILE)
+	@cp dockerfile-$(DISTRO_NAME).template $(DOCKER_FILE)
 	@source $(CONFIG) ; \
 	sed -i "s@__BASE_PACKAGES__@$${BASE_PACKAGES}@g" $(DOCKER_FILE) ; \
 	AT_PACKAGES="$${AT_RUNTIME_PACKAGES}" ; \
 	[[ "$(IMAGE_PROFILE)" == "devel" ]] && AT_PACKAGES="$${AT_PACKAGES} $${AT_DEVEL_PACKAGES}" ; \
-	sed -i "s@__AT_PACKAGES__@$$(echo " $${AT_PACKAGES}" | sed "s/  */ advance-toolchain-at\$${AT_VERSION}\$${EXTRA}-/g")@g" $(DOCKER_FILE)
+	sed -i "s@__AT_PACKAGES__@$$(echo " $${AT_PACKAGES}" | sed "s/  */ advance-toolchain-at\$${AT_VERSION}\$${EXTRA}-/g")@g" $(DOCKER_FILE);\
+    	sed -i "s@__DISTRO_REPO__@$${DISTRO_REPO}@g" $(DOCKER_FILE);\
+    	sed -i "s@__GPG_KEY__@$${GPG_KEY}@g" $(DOCKER_FILE)
```

Comment:
Please use the same style for the terminating ` ; \ ` as the previous lines, adding spaces before and after `;` . These lines should also be indented to the same level as the ones around it.

See my comment regarding `DISTRO_REPO` below.